uqcsbot/__init__.py

```diff
@@ -117,13 +139,19 @@ def main():
     else:
         # If in development mode, attempt to allocate an available bot token,
         # else stick with the default. If no bot could be allocated, exit.
-        bot_token = get_test_bot_token() if args.dev else SLACK_BOT_TOKEN
-        if bot_token is None:
-            print('Something went wrong during bot allocation. '
-                  'Please ensure there are bots available and try again later. '
-                  'Exiting.')
-            sys.exit(1)
-        bot.run(bot_token, SLACK_VERIFICATION_TOKEN)
+        if args.dev:
+            test_bot = get_free_test_bot()
+            if test_bot is None:
+                logger.error('Something went wrong during bot allocation. '
+                      'Please ensure there are bots available and try again later. '
+                      'Exiting.')
+                sys.exit(1)
+            token = get_bot_token(test_bot['user']['id'])
```

Comment:
`get_bot_token`is simple enough to just inline it. If it looks too confusing, just pull out the id as its own variable
i.e.
```
bot_id = test_bot['user']['id']
bot_token = BOT_TOKENS.get(bot_id, None)
```

be sure to handle if bot_token is `None` too